server/api/api/endpoints/nuclio.py

```diff
@@ -0,0 +1,22 @@
+import mlrun.common.schemas
+from mlrun.utils import logger
+import server.api.utils.clients.async_nuclio
+from server.api.api import deps
+from fastapi import (
+    APIRouter,
+    Depends,
+    Query,
+)
+
+router = APIRouter()
+
+
+@router.get("/nuclio/api-gateways")
+async def list_api_gateways(
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+    namespace=Query(default=None, alias="namespace"),
+    project_name=Query(default=None, alias="project-name"),
+):
+    return await server.api.utils.clients.async_nuclio.Client(
```

Comment:
you must verify authorization
you may do it with `server.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions`, action should be "read", resource type should be "api-gateways"